src/config/providers/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./Providers";
+export * from "./ReactQueryProvider";
+export * from "./SentryProvider";
```

Comment:
Shouldn't we only export `Providers`? I thought the idea was for the `providers` folder to encapsulate all the providers, and to only export the `Providers`